<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EMLR0Q3" failures="2" tests="6" name="Test" time="47.965" errors="0" timestamp="2025-02-20T00:24:34 IST">
  <testcase classname="testCases.TTC001_accountRegistrationTest" name="verify_account_registration" time="2.039"/>
  <testcase classname="testCases.TC002_loginTest" name="verify_login" time="1.608"/>
  <testcase classname="testCases.TC003_loginDataDrivenTest" name="verify_loginDataDrivenTest" time="1.603"/>
  <testcase classname="testCases.TC003_loginDataDrivenTest" name="verify_loginDataDrivenTest" time="10.111">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_loginDataDrivenTest.verify_loginDataDrivenTest(TC003_loginDataDrivenTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDataDrivenTest -->
  <testcase classname="testCases.TC003_loginDataDrivenTest" name="verify_loginDataDrivenTest" time="10.12">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_loginDataDrivenTest.verify_loginDataDrivenTest(TC003_loginDataDrivenTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDataDrivenTest -->
  <testcase classname="testCases.TC003_productSearch" name="productSearch" time="7.691"/>
</testsuite> <!-- Test -->
